tasks/deploy/config/mainnet/morpho-blue-wbtc-usdc-182day.ts

```diff
@@ -0,0 +1,140 @@
+import {
+    Address,
+    encodeAbiParameters,
+    keccak256,
+    parseEther,
+    toBytes,
+    zeroAddress,
+} from "viem";
+import {
+    HyperdriveInstanceConfig,
+    SIX_MONTHS,
+    USDC_ADDRESS_MAINNET,
+    WBTC_ADDRESS_MAINNET,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { MAINNET_FACTORY_NAME } from "./factory";
+import { MAINNET_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const MAINNET_MORPHO_BLUE_WBTC_USDC_182DAY_NAME =
+    "ElementDAO 182 Day Morpho Blue WBTC/USDC Hyperdrive";
+
+// USDC only has 6 decimals.
+const CONTRIBUTION = 100_000_000n;
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken: WBTC_ADDRESS_MAINNET,
+            oracle: "0xDddd770BADd886dF3864029e4B377B5F6a2B6b83" as `0x${string}`,
+            irm: "0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const MAINNET_MORPHO_BLUE_WBTC_USDC_182DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: MAINNET_MORPHO_BLUE_WBTC_USDC_182DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: keccak256(
+            toBytes(MAINNET_MORPHO_BLUE_WBTC_USDC_182DAY_NAME),
+        ),
+        salt: toBytes32("0x42080085"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        // NOTE: The latest variable rate on the Morpho Blue market is 2.93% APY:
```

Comment:
the latest rate isn't 2.93%. is this a copy paste thing?

Reply:
Yes it is totally a copy paste thing. Updated!